README.rst

```diff
@@ -10,3 +10,116 @@ flamingo
 
 Flamingo is a python3-based, `pelican <https://blog.getpelican.com/>`_-inspired
 static site generator.
+
+
+Basic Concepts
+--------------
+
+Flamingo aims to convert content written in a markup language into a static
+website. To archive this the flamingo core parses a directory of markup files
+and stores its content in an internal state called the ``context``. Plugins can
```

Comment:
Not sure if `context` should have code markup here. Isn't this here more referring to the 'concept' of a *context* than the actual potential code element? Maybe use `*context*` here instead.